pageserver/src/page_service.rs

```diff
@@ -1,13 +1,5 @@
-//
 //! The Page Service listens for client connections and serves their GetPage@LSN
 //! requests.
-//
-//   It is possible to connect here using usual psql/pgbench/libpq. Following
-// commands are supported now:
-//     *status* -- show actual info about this pageserver,
```

Comment:
'status' was removed in commit 9cce4304306b3d159ca442b30b97011bd4128822. And there are many other commands like 'basebackup' that were not listed here, so I just removed the obsolete comment.